spec/unit/classes/master/config_spec.rb

```diff
@@ -26,53 +26,53 @@
   context 'when PuppetDB and Puppet Master are on the same server' do
 
     context 'when using default values' do
-
       let(:pre_condition) { 'class { "puppetdb": }' }
 
       it { should contain_puppetdb_conn_validator('puppetdb_conn').with(
-        :puppetdb_server => 'puppetdb.example.com',
-        :puppetdb_port => '8081',
-        :use_ssl => 'true') }
+                    :puppetdb_server => 'puppetdb.example.com',
+                    :puppetdb_port => '8081',
+                    :use_ssl => 'true') }
     end
 
     context 'when puppetdb class is declared with disable_ssl => true' do
-
       let(:pre_condition) { 'class { "puppetdb": disable_ssl => true }' }
 
       it { should contain_puppetdb_conn_validator('puppetdb_conn').with(
-        :puppetdb_port => '8080',
-        :use_ssl => 'false')
-      }
-      
+                    :puppetdb_port => '8080',
+                    :use_ssl => 'false')}
     end
 
     context 'when puppetdb_port => 1234' do
-
       let(:pre_condition) { 'class { "puppetdb": }' }
-
       let(:params) do { :puppetdb_port => '1234' } end
 
       it { should contain_puppetdb_conn_validator('puppetdb_conn').with(
-        :puppetdb_port => '1234',
-        :use_ssl => 'true')
-      }
-
+                    :puppetdb_port => '1234',
+                    :use_ssl => 'true')}
     end
 
     context 'when puppetdb_port => 1234 AND the puppetdb class is declared with disable_ssl => true' do
-
       let(:pre_condition) { 'class { "puppetdb": disable_ssl => true }' }
-
-      let(:params) do {
-        :puppetdb_port => '1234'
-      } end
+      let(:params) do {:puppetdb_port => '1234'} end
 
       it { should contain_puppetdb_conn_validator('puppetdb_conn').with(
-        :puppetdb_port => '1234',
-        :use_ssl => 'false')
-      }
+                    :puppetdb_port => '1234',
+                    :use_ssl => 'false')}
 
```

Comment:
Not crititcal, but I might be tempted to leave out the whitespace changes or put them in a separate patch so that this one is focused on the /pdb/query changes.